docs/development/extensions-core/bloom-filter.md

```diff
@@ -154,25 +150,25 @@ for the query.
 }
 ```
 
-response
+Example response:
 
 ```json
-[{"timestamp":"2015-09-12T00:00:00.000Z","result":{"userBloom":"BAAAJhAAAA..."}}]
+[
+  {
+    "timestamp":"2015-09-12T00:00:00.000Z",
+    "result":{"userBloom":"BAAAJhAAAA..."}
+  }
+]
 ```
 
-These values can then be set in the filter specification described above.
-
-Ordering results by a bloom filter aggregator, for example in a TopN query, will perform a comparatively expensive
-linear scan _of the filter itself_ to count the number of set bits as a means of approximating how many items have been
-added to the set. As such, ordering by an alternate aggregation is recommended if possible.
+Ordering results by a Bloom filter aggregator, for example in a TopN query, can be resource-intensive. This is because the operation performs an expensive linear scan of the filter to approximate the count of items added to the set by counting the number of set bits. We recommend ordering by an alternative aggregation method.
```

Comment:
```suggestion
We recommend ordering by an alternative aggregation method instead of ordering results by a Bloom filter aggregator.
Ordering results by a Bloom filter aggregator can be resource-intensive because Druid performs an expensive linear scan of the filter to approximate the count of items added to the set by counting the number of set bits. 
```
Suggest putting the recommendation first. Then the explanation.